### DISCUSSION TITLE
Can History State Node be an Initial State ?

### DISCUSSION THREAD

**Discussion by nnduc1994** [statelyai/xstate#2001] [Q&A]
As the title, can History State Node be an Initial State ( with the help of target when define the state ) . 
Code Example: https://xstate.js.org/viz/?gist=d5145bd45c764c5b95bc7371e0b06c87

I tried it out, when starting the machine, the machine stuck in the history state instead of going to its target

**Comment by Andarist** [comment]
It should be possible as per SCXML spec:
> If a transition is executed that takes the <history> state as its target, the behavior depends on whether the parent state has been visited before. If it has, the SCXML processor must behave as if the transition had taken the stored state configuration for that history state as its target. If it has not, the SCXML processor must behave as if the transition had taken the default stored state configuration for that history state as its target.

if it doesnt work then please create an issue for it

**Reply by nnduc1994** [reply]
Opened the issue https://github.com/davidkpiano/xstate/issues/2002
